Better Call Saul: Lawyer Agency

EdwardNashton opened the pull request:
Replaced and remade Lawyer Agency. 2 floors, offices and stuff.

![image](https://user-images.githubusercontent.com/19852466/166128987-054bc875-dadd-4482-b7f6-1384c81a0e54.png)

![image](https://user-images.githubusercontent.com/19852466/166128998-15b7ec35-e87f-4a50-b508-10434fb867d0.png)

Slightly remade a Living Block (ghetto) power substation nearby.

![image](https://user-images.githubusercontent.com/19852466/166129010-6172bc29-7ab9-488c-bf42-2a3af3903a80.png)

Expanded a road on a place where previous version of Agency were. There will be a way to a new Mine, that I need to restore, as this office, but it will be in a next PR.

![image](https://user-images.githubusercontent.com/19852466/166129026-251dad1d-d501-4319-8668-037cc8569eed.png)

Commit:
Better Call Saul: Lawyer Agency

Replaced and remade Lawyer Agency. 2 floors, offices and stuff.
Slightly remade a Living Block (ghetto) power substation nearby.
Expanded a road on a place where previous version of Agency were.

ProfessorPopoff commented:
Looks very nice. Ignore the linters error, minor unrelated problem. 

Two comments.

![image](https://user-images.githubusercontent.com/51188571/166129198-7a5adc9f-e104-4486-b3b1-553cebdd37bf.png)
You removed a wastelander spawn from this road. Also it leading into nothing is weird. I see you gave a reason for it, and I believe you'll follow up on it, but it'd be bad practice to put their very off road here pre-emptively. Also the rest of the map's intersections (more or less) looks like this, and should probably be kept consistent. I mostly mean the lines, as seen here. 
![image](https://user-images.githubusercontent.com/51188571/166129251-ee1d4daf-ff50-4a25-a228-5edd05a1748e.png)

The light setup in the main office is really weird and inconsistent. You should try to even it out a bit
![image](https://user-images.githubusercontent.com/51188571/166129270-ea8b2840-8088-4194-b932-fdfeb8c4d6e2.png)

Also I don't know what's up but this guy got a crazy 3 piece setup going on here. 
![image](https://user-images.githubusercontent.com/51188571/166129305-8967c7be-aa20-4b9a-98e6-c15a691c8138.png)

Those first two are the main concerns, address them and I can merge it. Thanks again, looks nice.

Commit:
Better Call Saul: fixes.

Lightning on a second floor. Safe is under the flag.
The road is under constructionf, for now.

ProfessorPopoff commented:
It's ok... Merging now.
